Kvent/admin.py

```diff
@@ -1,12 +1,6 @@
 from django.contrib import admin
-from .models import Event
+from .models import *
 
-class EventAdmin(admin.ModelAdmin):
-    list_display = (
-        'event_name',
-        'location',
-        'short_description'
-    )
-
-
-admin.site.register(Event, EventAdmin)
```

Comment:
You should keep these and add the display interface in the admin page to User too!
something like `class UserAdmin(admin.ModelAdmin):
    list_display = (
        'username',
        'email'
    )`